### DISCUSSION TITLE
web

### DISCUSSION THREAD

**Discussion by XxshurikenxXx** [community/community#28590] [Other Feature Feedback, Questions, & Ideas]
hello guys I have had a problem with the github.page I wanted to configure a page I have followed many tutorials to the letter and when I want to access the page it leaves me it says that there is no page I have been using the following repository
https://github.com/SupunKavinda/jekyll-theme-leaf

**Comment by radar-blip** [comment]
index.md should be index.html probably, try it.

**Comment by jfullstackdev** [comment]
you already have the page

![image](https://user-images.githubusercontent.com/47092464/183275591-685cdc3e-92d4-48c9-9519-15dd98ae3947.png)

but as you can see, it's not at your root, 

you should rename the repo `Hacking-0Zone.github.io`
